lib/Whatbot/Command/Role/Location.pm

```diff
@@ -0,0 +1,88 @@
+###########################################################################
+# Location.pm
+# the whatbot project - http://www.whatbot.org
+###########################################################################
+
+use Moops;
```

Comment:
what is this, 2005?

Reply:
Is it not?

Reply:
In all seriousness, though, Moops was the closest thing at the time to something usable, without the inherent issues in MooseX::Declare. Moops is now stuck in a cycle of will-never-be-stable, but I probably wouldn't want to change anything until [Cor](https://gist.github.com/Ovid/68b33259cb81c01f9a51612c7a294ede) does something useful, or this gets rewritten in something else.

Reply:
I don't know anything about Moops aside from what I knew ten years ago. I just laugh every time I read the name.